### ISSUE TITLE
Exit code 11 ?

### ISSUE THREAD

**Issue by riahc3** [ryancbutler/Citrix-VAD-LAB#2]
Hello

Im trying to install a Delivery Controller and it seems it has given me a exit code of 11?

{
    "stdout": "parsing:\tC:\\ProgramData\\Citrix\\CTQs\\metainstaller.dat\\tmo.8aa9ef23-3c9a-4d80-b7e7-ebb1cd77d47f -done:1f3368e8-c140-4759-ac20-0594c7d84d00\r\nparsed:\t 1/1 in C:\\ProgramData\\Citrix\\CTQs\\metainstaller.dat\\tmo.*\r\nUnknown error FailedMedia.\r\n",
    "stderr": "",
    "changed": false,
    "rc": 11,
    "reboot_required": false,
    "msg": "unexpected rc from install  C:\\ProgramData\\Citrix\\XenDesktopSetup\\XenDesktopServerSetup.exe: see rc, stdout and stderr for more details",
    "stdout_lines": [
        "parsing:\tC:\\ProgramData\\Citrix\\CTQs\\metainstaller.dat\\tmo.8aa9ef23-3c9a-4d80-b7e7-ebb1cd77d47f -done:1f3368e8-c140-4759-ac20-0594c7d84d00",
        "parsed:\t 1/1 in C:\\ProgramData\\Citrix\\CTQs\\metainstaller.dat\\tmo.*",
        "Unknown error FailedMedia."
    ],
    "stderr_lines": [],
    "_ansible_no_log": false
}

I THINK this error code states that the installation media cannot be found. This happens if you, for example, mount a ISO and install from a ISO....Reboot comes along and ISO is not mounted....

Just a heads up, just in case

**Comment by ryancbutler** [comment]
Are you running from a UNC path?  I have had odd issues if the ISO becomes corrupted for some reason as well.  I would check the ISO with the checksum or redownload and try again.

**Comment by riahc3** [comment]
> Are you running from a UNC path? I have had odd issues if the ISO becomes corrupted for some reason as well. I would check the ISO with the checksum or redownload and try again.

No, I actually copied the ISO over to the (in this case) Delivery Controller.

You know what happened? I copied the ISO over, mounted the ISO and then ran the setup....It needs to reboot (at least) once so when it tried to start setup again, the ISO was not mounted so it did not continue.

In order to work around this, there are two options (I personally choose the latter)

1: After the reboot, remount the ISO and then call the setup in ProgramData
2: Extract the ISO completely and run it from the folder you extracted it in. This means the files will always be avaliable.
